ejb/src/main/java/biz/karms/sinkit/ejb/impl/ArchiveServiceEJB.java

```diff
@@ -105,6 +105,12 @@ public boolean setVirusTotalReportToIoCRecord(final IoCRecord ioc, final IoCViru
         return elasticService.update(ioc.getDocumentId(), vtReports, ELASTIC_IOC_INDEX, ELASTIC_IOC_TYPE, null);
     }
 
```

Comment:
This should only accept IoCAccuCheckerReport to guarantee that hashes do not need to be recomputed.